src/index.html

```diff
@@ -15,8 +15,77 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo__img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="/apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+            <a
+              href="/samsung"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+            <a
+              href="/smartphones"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+            <a
+              href="/laptops & computers"
```

Comment:
Don't use spaces in the `href` attribute of the `<a>` tag. Replace spaces with hyphens or underscores to make the URL valid.